syncd/MdioIpcServer.cpp

```diff
@@ -56,6 +50,11 @@ MdioIpcServer::MdioIpcServer(
 
     /* globalContext == 0 for syncd, globalContext > 0 for gbsyncd */
     MdioIpcServer::m_syncdContext = (globalContext == 0);
+#ifdef MDIO_ACCESS_USE_NPU
+    MdioIpcServer::m_accessUseNPU = true;
```

Comment:
Why not directly `m_accessUseNPU = true` ? It is not defined as a static class member.

Since member `m_accessUseNPU` is added, macro `MDIO_ACCESS_USE_NPU` looks redundant and needs to be removed. Its value can be assigned by adding a parameter like `accessUseNPU` in the constructor.

Reply:
MDIO_ACCESS_USE_NPU is only defined in syncd/Malefile.am for "make configure PLATFORM=broadcom". It is only tested on broadcom ASIC. When other platforms have tested the feature, the syncd/Makefile.am can be modified to add the feature. Actually, if m_accessUseNPU = true for vs platform, we see vssyncd test logs error messages.

Reply:
The value of `m_accessUseNPU`  has indicated feature enablement, which could be TRUE if 

1. platform support, currently only broadcom
2. mock MdioIpcServer instance in unittest

Reply:
Yes, The value of m_accessUseNPU is TRUE if platform support, currently only broadcom. For mock MdioIpcServer instance in unittest, the value of m_accessUseNPU is changed to be true at run time by the function MdioIpcServer::setIpcTestMode().